dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.anno-2.0.feature

```diff
@@ -0,0 +1,22 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.anno-2.0
+IBM-API-Package: jakarta.annotation; type="spec", \
+ jakarta.annotation.security; type="spec", \
+ jakarta.annotation.sql; type="spec"
+IBM-SPI-Package: \
+ com.ibm.wsspi.anno.classsource, \
+ com.ibm.wsspi.anno.info, \
+ com.ibm.wsspi.anno.service, \
+ com.ibm.wsspi.anno.targets, \
+ com.ibm.wsspi.anno.util
+Manifest-Version: 1.0
+IBM-Process-Types: server, \
+ client
+-features=com.ibm.websphere.appserver.artifact-1.0, \
+ com.ibm.websphere.appserver.jakarta.annotation-2.0
+-bundles=com.ibm.ws.anno
+-jars=com.ibm.websphere.appserver.spi.anno; location:=dev/spi/ibm/
+-files=dev/spi/ibm/javadoc/com.ibm.websphere.appserver.spi.anno_1.1-javadoc.zip
```

Comment:
I don't know much about javadoc in the product, but does this need to be updated to anno_2.0 as well?

Reply:
No, a new version of the anno SPI is not being provided.  The anno SPI remains unchanged at version 1.1.  When looking at `wlp/dev/spi/ibm` and `wlp/dev/spi/ibm/javadoc` there will still only be one JAR with the SPI and one javadoc.  Specifically:
- com.ibm.websphere.appserver.spi.anno_1.1.40.jar
- com.ibm.websphere.appserver.spi.anno_1.1-javadoc.zip

The only difference with the anno-2.0 feature is that it makes `jakarta.annotation` public rather than `javax.annotation`.  There is no change to the anno SPI.

Reply:
Thy new feature name `com.ibm.websphere.appserver.anno-2.0` is called `2.0` to be consistent with other feature upgrades for Jakarta, and also to enable other features that use the anno SPI but don't care about javax vs jakarta.... they can use `tolerates` and tolerate 1.0 and 2.0.

Reply:
Thanks for the clarification.